### PR TITLE
[REVIEW] @Skalina review on generic component list

### PR DESCRIPTION
**Pull request by LordWeedlle**
https://github.com/rgsystemes/kb/issues/4613

### CODE REVIEW

File: src/Molecules/List/ListItem/ListItem.jsx [hunk 1]
```diff
@@ -1,12 +0,0 @@
-import styled from 'styled-components'
```
Comment (monzey) [inline src/Molecules/List/ListItem/ListItem.jsx]: why ?
When I click on the items now,  I have the ripple. I don't want that

Comment (LordWeedlle) [reply src/Molecules/List/ListItem/ListItem.jsx]: Can you show me where on the documentation you find the ripples options I removed?

File: src/Atoms/Popover/Popover.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import BasePopover from '@material-ui/core/Popover'
+import { css } from '@styled-system/css'
+import { layout } from 'styled-system'
+
+const Popover = styled(BasePopover)`
+  .MuiPopover-paper {
+    ${css({ border: '1px solid', borderColor: 'lightgrey', maxHeight: '440px' })};
```
Comment (mesteche) [inline src/Atoms/Popover/Popover.jsx]: The css interpreter is overkill here, all the values are plain css, nothing from the theme or the props:
```suggestion
    border: '1px solid';
    borderColor: 'lightgrey';
    maxHeight: '440px';
```

File: src/Molecules/List/ListItem/ListItem.jsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+const ListItem = ({ button = false, disableRipple = false, disableTouchRipple = false, ...props }) => {
+  if (button) {
+    disableRipple = true
+    disableTouchRipple = true
+  }
 
-ListItem.defaultProps = {
-  disableRipple:      true,
-  disableFocusRipple: true,
+  return <BaseListItem button={button} disableRipple={disableRipple} disableTouchRipple={disableTouchRipple} {...props}/>
```
Comment (mesteche) [inline src/Molecules/List/ListItem/ListItem.jsx]: Oh come on...
```suggestion
  return <BaseListItem
    button={button}
    disableRipple={!!button || disableRipple}
    disableTouchRipple={!!button || disableTouchRipple}
    {...props}
  />
```

**Pull request merged**